src/routes/(infoPages)/[topTierPage]/[...serviceGroupPage]/+page.server.ts

```diff
@@ -26,6 +106,7 @@ const query = gql`
 `;
 
 export const load = async ({ parent, params }) => {
+  if (!CONTENTFUL_SPACE_ID || !CONTENTFUL_DELIVERY_API_TOKEN) return ServiceGroupPageTestContent;
```

Comment:
@LouisFettet I saw that you removed most of the the test content in your nav work. Is this still needed?